scripts/rmvers_config.py

```diff
@@ -0,0 +1,2 @@
+c.Exporter.preprocessors = ['scripts.removeversion.RemoveVersionPreprocessor']
```

Comment:
I think it's easier to add the `ClearOutputPreprocessor` here than to call nbconvert twice. It's also faster (fewer disk reads/writes).